src/components/RegistrationQuestionsPage/RegistrationQuestionsPage.css

```diff
@@ -5,9 +5,3 @@
 .registration-questions-page__create-form {
   padding: 0.5rem 1rem 1rem 1rem;
 }
-
-.question-set-list {
-  list-style: none;
-  margin: 0;
-  padding: 0;
-}
```

Comment:
Moved this to `RegistrationQuestionSetList.css` and `PeerReviewQuestionSetList.css`